mysys/crc32/crc32_arm64.c

```diff
@@ -2,10 +2,20 @@
 #include <string.h>
 #include <stdint.h>
 
-#if defined(__GNUC__) && defined(HAVE_ARMV8_CRC)
+#if defined(__GNUC__) && defined(HAVE_ARMV8_CRC) || defined(__FreeBSD__) && __FreeBSD_version >= 1200000
```

Comment:
have_armv8_crc (for freebsd) should be enabled based on the cmake check above.
so ideally we can just keep the freebsd check limited to the getauxval only ?
[unless cc compiler (whatever is the default in freebsd) doesn't define __GNUC__ flag].